README.md

```diff
@@ -1,17 +1,17 @@
-# blindnet devkit documentation
+# blindnet.dev website
 
 <p align="center">
-  <strong>Documentation website for the blindnet devkit.</strong>
+  <strong>A complete portal helping developers build trustworthy applications through privacy..</strong>
```

Comment:
```suggestion
  <strong>A portal helping developers build trustworthy applications through privacy..</strong>
```
I think developers will see for themsleves that the portal is complete. It is a much stronger message if we are not the ones telling it :)